Prison Station Rework

Hathkar opened the pull request:
<!-- ***STOP!***  Read this: If this is not a PR ready for review and merge or WIP, open it as a draft PR, using the arrow next to 'Create Pull Request'>

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Added ceilings to every room in prison station.
Max security and research in the north are now considered underground.
Medium Security and the crashed ship in the south are now considered underground.
Various mining drills have been placed around the map.
Platinum drills are located in the north and south, with one in the yard.
Added some rock walls and coastline around  the underground areas to help visually signify their locations.
Added more survivor spawn locations
Adjusted some of the starting resin node locations in the north and south.
Moved the resin silo from below LZ1 to north research.
Redesigned LZ2 to not be hot garbage, and actually worth using.
Removed the timed shutters located outside the Landing Zones.

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: More survivor spawn points in Prison Station
add: More mining drills in Prison Station
add: Rock walls and coastline in Prison Area to make "underground" areas visually identifiable
balance: Made LZ2 actually worth using as a FOB
del: Removed timed shutters outside LZs in Prison Station
tweak: Resin node locations
tweak: Moved the LZ1 resin silo to the north Research Area
code: Added ceilings to each area in Prison Station
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Hughgent commented:
You've added the `ceiling` var to almost every typepath in the areas.dm file.

You don't need to do this. If you added it once to the 'parent' then it will propagate to all the children.

Please remove the extraneous `ceiling` vars.

Hathkar commented:
> 
> 
> You've added the `ceiling` var to almost every typepath in the areas.dm file.
> 
> You don't need to do this. If you added it once to the 'parent' then it will propagate to all the children.
> 
> Please remove the extraneous `ceiling` vars.

Done! Added some new parent types, and removed extra ceiling vars.
